tools/stress-test/src/main.rs

```diff
@@ -0,0 +1,534 @@
+use std::cmp;
+use std::collections::HashMap;
+use std::path::PathBuf;
+use std::sync::atomic::{AtomicU32, AtomicU64};
+use std::sync::{Arc, Mutex};
+use std::time::Instant;
+
+use clap::{Args, Parser, Subcommand};
+
+use ockam::abac::tokio::runtime::Runtime;
+use ockam::compat::tokio;
+use ockam::compat::tokio::task::JoinSet;
+use ockam::{Context, NodeBuilder, TcpListenerOptions, TcpTransport};
+use ockam_api::nodes::service::{NodeManagerGeneralOptions, NodeManagerTransportOptions};
+use ockam_api::nodes::{InMemoryNode, NodeManagerWorker, NODEMANAGER_ADDR};
+use ockam_api::CliState;
+use ockam_core::Address;
+
+use crate::config::Config;
+use crate::portal_simulator::PortalStats;
+
+mod config;
+mod portal_simulator;
+
+#[derive(Debug, Args, Clone)]
+struct RunCommand {
```

Comment:
It would be good to split this file into several concerns:

 - Making commands
 - Executing them
 - Display data
 - ...

Reply:
I split the logic of main into several files: main, display, execution and stats.